### PR TITLE
generating 5.1.8 release notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_5.1.md [hunk 1]
```diff
@@ -19,6 +19,50 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+
+- **I95-41108 EthoSVR doesn't restart the session when all peer paths are disabled:** This is not specific to EthoSVR and can happen to any session that is unidirectional. This issue has been resolved. 
+------
+- **I95-41802/I95-43264 CVE Issues:** The latest Security vulnerabilities have been identified and addressed.
+------
+- **I95-41876/I95-43624 CVE on CentOS 7:** The latest CentOS 7 security vulnerabilities have been identified and addressed.
+------
+- **I95-42364 `off-subnet-reverse-arp-mac-learning` does not work when `outbound-only` is configured:** The learned MAC is now saved in the reverse flow placeholder for use when the flow is established. 
+------
+- **I95-43066 Issue with database query co-ordinator:** Resolved an issue that when the buffer queue is full, the message is dropped, but the event was not handling the inflight message.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:40]: co-ordinator is not hyphenated. The process name within the system is `databaseQueryCoordinator`, not that that matters to customers.
Suggested heading: "Requests for metrics are timing out"

File: docs/release_notes_128t_5.1.md [hunk 2]
```diff
@@ -19,6 +19,50 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-43389 SALT Minion file truncation:** This issue has been resolved.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:42]: salt is not in all caps (https://saltproject.io/)

File: docs/release_notes_128t_5.1.md [hunk 3]
```diff
@@ -19,6 +19,50 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-43454 Plugin highstate not sync'd:** When applying DNS cache configuration changes via PCLI or GUI, highstate is not being applied from the conductor to router. This issue was resolved by removing support of legacy pillar generation for plugins.
+------
+- **I95-43591 When deleting a linux user, the default bash is not restored:** A protection has been added to the SSR to disallow the creation of a user that already exists on the linux system.
+------
+- **I95-43604 NAT keep alive problem:** Resolved an issue where the keep-alive cache entry was being removed when the flow was invalidated.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.1.md:48]: keepalive in the domain of software algorithms is usually one word or hyphenated.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_5.1.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md:40]: The first of the cut and paste errors. Updated.

File: docs/release_notes_128t_5.1.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md:42]: Fixed

File: docs/release_notes_128t_5.1.md (hunk 3 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md:48]: fixed

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**